class/defaults.yml

```diff
@@ -57,6 +57,10 @@ parameters:
         enabled: false
         secretNamespace: ${crossplane:namespace}
         postgres:
+          # bucket_region: 'lpg' || 'ch-gva-2'
+          bucket_region: ''
+          # bucket_endpoint: 'https://objects.lpg.cloudscale.ch' || 'https://sos-ch-gva-2.exo.io'
+          bucket_endpoint: ''
```

Comment:
Please fail compilation if these aren't set.

Reply:
I added assertions but now more things fails, it works when You isolate test case:
`make test -e instance=vshn` 

but fails when `make gen-golden-all` is run

Reply:
That's because you assert this unconditionally. You need to only assert this if PostgreSQL by VSHN is actually enabled

Reply:
I would also consider to create an object `backup` where all backup info would reside. Also consider if we can enable or disable backups altogether.

Reply:
I agree, but if we actually have at least one use case of postgresql without backup

Reply:
We probably can't disable backups easily without any conditionals in the composition. Let's stick with fix enabled backups for now.

Reply:
Fine by me.